liquidcore/home/signals.py

```diff
@@ -9,7 +9,8 @@
 from django.db.models.signals import (post_save,
                                       pre_save,
                                       pre_delete,
-                                      m2m_changed)
+                                      m2m_changed,
+                                      post_delete)
```

Comment:
at 5-6 imports from the same module, i think it's time to refactor the code to do `import django.db.models.signals as signals` and use `signals.post_delete` everywhere.